_rules/audio-transcript-2eb176.md

```diff
@@ -24,12 +31,12 @@ acknowledgments:
 
 This rule applies to every [non-streaming](#non-streaming-media-element) `audio` element that is:
 
-- playing; or,
-- has a [play button][] that is [visible][] and [included in the accessibility tree][].
+- autoplaying; or,
+- has a [play button][] that is [visible][].
 
 ## Expectation
 
-The auditory information of each test target is available through a text transcript. That text transcript is [visible][] and [included in the accessibility tree][], either on the page or through a link.
+The auditory information of each test target is available through a text transcript. That text transcript is [visible][], either on the page or through a clearly labeled location.
```

Comment:
I'm not sure why it is OK to have a transcript that is not included in the accessibility tree.
Wouldn't that be problematic, e.g., for Braille display users?

Reply:
I believe it was due to the fact the transcript might not be included in the current page and therefore not in the accessibility tree.